packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/_tests/RestaurantRating.test.js

```diff
@@ -158,6 +158,54 @@ describe('RestaurantRating component', () => {
         });
     });
 
+    describe('reviews count threshold', () => {
+        const countThreshold = 200;
+        it('shows proper value when count is less than threshold', () => {
```

Comment:
We should try to make the test names more descriptive rather than saying 'proper value'